src/Test/Utilities/Portable/Compilation/TestSyntaxTreeOptionsProvider.cs

```diff
@@ -26,17 +27,27 @@ public sealed class TestSyntaxTreeOptionsProvider : SyntaxTreeOptionsProvider
                     x => x.Item2,
                     comparer)
             );
+            if (globalOption.Item1 is object)
```

Comment:
Can we instead make globalOption a `(string, ReportDiagnostic)` to make the null handling clearer? This file is #nullable enabled so presumably you shouldn't be able to get a null first parameter; the fact you can pass default in line 50 is a compiler bug: https://github.com/dotnet/roslyn/issues/38957